indicatorStatusIcon.js

```diff
@@ -116,7 +120,9 @@ class AppIndicators_IndicatorStatusIcon extends PanelMenu.Button {
         this._updateStatus();
         this._updateMenu();
 
-        Main.panel.addToStatusArea("appindicator-"+this._indicator.uniqueId, this, 1, 'right')
+        let our_id = "appindicator-"+this._indicator.uniqueId;
```

Comment:
Use <code>\`appindicator-${this._indicator.uniqueId\`</code> instead

Reply:
Well, this (old line 119) has been introduced with commit 5289ceb, and I just kept it.
But I changed it now.